### PR TITLE
Fix Bug 1175439, cleanup l10n tags on plugincheck page

### PR DESCRIPTION
**Pull request by schalkneethling**
@flodolo r? I took a stab at this, did I understand what you meant correctly? Thanks!

### CODE REVIEW

File: bedrock/mozorg/templates/mozorg/plugincheck.html [hunk 1]
```diff
@@ -74,13 +74,13 @@
...
   {% set share_urls = { 'twitter': 'http://mzl.la/17aqnLZ', 'googleplus': 'http://mzl.la/1FuAhTS', 'facebook': 'http://mzl.la/1z4Noqm' } -%}
-  {% if l10n_has_tag('share_cta') -%}
+  {% if l10n_has_tag('plugincheck_nosupport') -%}
     {% set share_text = _('Plugins don’t always update automatically. Check yours now to stay safe and browse without interruption!') %}
   {%- else -%}
     {# If the new copy is not localized yet, just use the same text as the existing preamble #}
     {% set share_text = _('Keeping your plugins up to date helps Firefox run safely and smoothly.') %}
   {% endif -%}
-  {{ share_cta(_('Share'), share_urls, share_text, 'spread-plugincheck', 'sky mini') }}
+  {{ plugincheck_nosupport(_('Share'), share_urls, share_text, 'spread-plugincheck', 'sky mini') }}
```
Comment (flodolo) [inline bedrock/mozorg/templates/mozorg/plugincheck.html]: You don't want to change this ;-)

**Comment by flodolo** [comment]
Besides one unrelated change, it looks good, thanks.

File: bedrock/mozorg/templates/mozorg/plugincheck.html (hunk 1 above)
Comment (schalkneethling) [inline bedrock/mozorg/templates/mozorg/plugincheck.html]: Yeah, that was my initial feeling as well :)

Comment (schalkneethling) [inline bedrock/mozorg/templates/mozorg/plugincheck.html]: Oh wait, find and replace fail :blush:

**Comment by schalkneethling** [comment]
@flodolo fixed and squashed

**Pull request merged**